plugins/generator-1.21.4/neoforge-1.21.4/templates/elementinits/menus.java.ftl

```diff
@@ -67,22 +67,15 @@ public class ${JavaModName}Menus {
 	     */
 	    updateMenuState(entity, elementType, name, elementState); //This method will also be called from the network packet on the opposite side.
 	    if (entity.level().isClientSide) {
-	        ${JavaModName}Screens.onMenuStateUpdate(elementType, name, elementState); //A temporary method for updating GUI elements, after adding synchronization
-	        //it will be called from the packet received by the client from the server.
+	        ${JavaModName}Screens.onMenuStateUpdate(elementType, name, elementState);
 	    }
 	}
 
     <#-- At the moment this getter method returns a value only from the called side, it is not synchronized with the opposite side. -->
-	public static <T> T getMenuState(Entity entity, int elementType, String name, T defaultValue) {
+	public static <T> T getMenuState(Entity entity, String elementType, String name, T defaultValue) {

```

Comment:
Why is this string again, we agreed you will use int since you will need this data in the packets, right?

You can also contact int and string ;)

Reply:
This is exactly what I meant and I don't understand why you always change type when I ask you this.

It is perfectly legal to concatenate int and String to make e.g. 0:name and 1:name